test/integration/util/util.go

```diff
@@ -101,3 +88,29 @@ func StartScheduler(clientSet clientset.Interface, enableEquivalenceCache bool)
 	}
 	return schedulerConfigurator, shutdownFunc
 }
+
+// createConfigurator create a configurator for scheduler with given informer factory and default name.
+func createConfigurator(
```

Comment:
Since this is in test utils (as opposed to scheduler test utils), I think you should rename it to createSchedulerConfigurator.

Reply:
Yep, just fixed.